docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst

```diff
@@ -310,6 +327,23 @@ approach is twofold:
     provides a way to directly model these effect sizes in a way that preserves
     this interpretation.
 
+.. list-table:: Prevalence of Vitamin A deficiency in GBD 2017
+  :widths: 15 14 14 
+  :header-rows: 1
+
+  * - Source
```

Comment:
There is no actual data in the gbd mapping, just metadata (like ids and distribution type).  GBD compare is a good place to look, but for vitamin a, there are two potential draw level sources we might use: prevalence from `como` using cause id 389 or exposure for the `exposure` source using rei_id 96.

Reply:
Currently it is unclear what's in this table.

Reply:
Agreed, we should clarify where the data in the second column is coming from!

Reply:
This table aims to compare prevalence data from two sources (como using cause id 389 and GBD compare). They are both from Como. I will replace the GBD compare data with the exposure source using rei_id 96.